src/tcgwars/logic/impl/gen8/ChillingReign.groovy

```diff
@@ -1868,13 +1872,13 @@ public enum ChillingReign implements LogicCardInfo {
               delayed {
                 before PLAY_ENERGY, {
```

Comment:
`PLAY_ENERGY` won't prevent the attachments properly, `ATTACH_ENERGY` should be checked instead (See: Hunchkrow-GX; a few cards in other sets seem to be using "play", may need to be tested/fixed)